modules/extensions-xml-support/src/test/resources/modules/nested/module-global-element-another-proxy.xml

```diff
@@ -13,7 +13,9 @@
     <property name="somePassConfig" type="string" defaultValue="some-password"/>
     <property name="port" type="string"/>
 
-    <module-global-element-proxy:config name="another-proxy-config" someUserConfig="#[properties.someUserConfig]" somePassConfig="#[properties.somePassConfig]" port="#[properties.port]"/>
+    <module-global-element-proxy:config name="another-proxy-config" someUserConfig="#[properties.someUserConfig]" somePassConfig="#[properties.somePassConfig]" port="#[properties.port]" >
+        <module-global-element-proxy:connection />
```

Comment:
Will this element contain all the parameters of the module that are related to the connection?. That would make it consistent with what we have in the Java connectors

Reply:
left for other JIRA https://www.mulesoft.org/jira/browse/MULE-13409